src/com/cdd/bao/template/SchemaVocab.java

```diff
@@ -304,6 +327,15 @@ public static SchemaVocab deserialise(InputStream istr, Schema[] templates) thro
 			sv.treeList.add(stored);
 		}
 		
+		int nRemappings = data.readInt();
```

Comment:
Follow the same variable naming style as the rest of the method (i.e. not capitalising). This isn't a global rule, it's just a regional aesthetic consistency.